app/bank/domain/historic_movement_repository.py

```diff
@@ -1,9 +1,19 @@
 from abc import abstractmethod, ABC
+from datetime import datetime
+from typing import Optional
+from uuid import UUID
+
+from django.db.models import Q
 
 from bank.domain.historic_movement import HistoricMovement
 
 
 class HistoricMovementRepository(ABC):
+    @abstractmethod
+    def filter_movement(self, source_account:Optional[UUID] = None, created_at:Optional[datetime] = None):
```

Comment:
missing return value

Also the naming suggest it will return a single element. This will probably return a List